src/s2b_tent_city_chase.tw2

```diff
@@ -0,0 +1,530 @@
+::Tent City - Intro
+I tore through a patched seam in the back of mom’s tent and felt a twinge of guilt as I broke through. Mom would understand, but Ron would be sure to give her hell while he fixed the busted tarp for the umpteenth time.
```

Comment:
Mom's Tent scene ends with the PC tearing the tarp, so we should remove "tore through a patched seam in the back of mom's tent and" and maybe add "the tarp" after "through.